### PR TITLE
Show dream orbs in search

### PR DESCRIPTION
**Pull request by DraKay**
I hope it works... It should show dream orbs page in the search bar

### CODE REVIEW

File: scripts/typeahead.js [hunk 1]
```diff
@@ -161,6 +161,12 @@ const searchOptions = [
     type: 'Pokérus',
     page: '',
   },
+  // Dream Orbs
+  {
+    display: 'Dream Orbs',
+    type: 'Dream Orbs',
```
Comment (RedSparr0w) [inline scripts/typeahead.js:167]: This is the correct way of adding it, but you will need to remove the space as the page is `DreamOrbs` or rename the folders to be `Dream Orbs`
```suggestion
    display: 'DreamOrbs',
    type: 'DreamOrbs',
```

Comment (PixLSteam) [reply scripts/typeahead.js:167]: I think the folder should be `Dream Orbs`. The other pages like Hatchery Helpers also use spaces.

Comment (DraKay) [reply scripts/typeahead.js:167]: I don't think I can change the folder name easily from my device...

Comment (RedSparr0w) [reply scripts/typeahead.js:167]: No worries, i'll update it into this PR

**Review by RedSparr0w — APPROVED** [review]
Looks good, thanks for the PR! 👍

**Pull request merged**